presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -108,6 +108,21 @@ The properties that apply to Hive connector security are listed in the
 :doc:`/connector/hive-security` section for a more detailed discussion of the
 security options in the Hive connector.
 
+HDFS Permissions
+^^^^^^^^^^^^^^^^
+Before running any ``CREATE TABLE`` or ``CREATE TABLE ... AS`` statements
+for Hive tables in Presto, you need to check that the operating system user
+running the Presto server has access to the Hive warehouse directory on HDFS. The Hive warehouse
+directory is specified by the configuration variable ``hive.metastore.warehouse.dir``
+in ``hive-site.xml``, and the default value is ``/user/hive/warehouse``. If that
+is not the case, either add the following to ``jvm.config`` on all of the nodes:
```

Comment:
This content is very useful, but some of it is redundant with the above "HDFS Username" section. Can you merge these two sections together? Maybe titling it "HDFS Username and Permissions"?